R/aedseo.R

```diff
@@ -32,6 +36,7 @@
 #'   disease threshold?
 #'   - 'seasonal_onset_alarm': Logical. Is there a seasonal onset alarm?
 #'   - 'converged': Logical. Was the IWLS judged to have converged?
+#'   - 'skipped_window': Logical. Was the window skipped?
```

Comment:
Can you specify why it would be skipped?

e.g. - 'skipped_window': Logical. Was the window skipped due to missing data?